pkgs/tools/audio/beets/default.nix

```diff
@@ -154,7 +157,8 @@ in pythonPackages.buildPythonApplication rec {
     ++ optional enableMpd           pythonPackages.mpd2
     ++ optional enableSonosUpdate   pythonPackages.soco
     ++ optional enableThumbnails    pythonPackages.pyxdg
-    ++ optional enableWeb           pythonPackages.flask
+    ++ optional (enableWeb
+              || enableZero)        pythonPackages.flask
```

Comment:
Just a **very** pedantic nitpick (feel free to ignore as you please):

```suggestion
    ++ optional (enableWeb || enableZero) pythonPackages.flask
```

... plus changing all the other lines as well, so that in the end it looks like this (minus the first line of course):

```nix
{x=[ # <- just to avoid weird syntax highlighting :-)

  ] ++ optional enableAbsubmit            essentia-extractor
    ++ optional enableAcoustid            pythonPackages.pyacoustid
    ++ optional (enableFetchart
              || enableEmbyupdate
              || enableKodiupdate
              || enableLoadext
              || enablePlaylist
              || enableSubsonicupdate
              || enableAcousticbrainz)    pythonPackages.requests
    ++ optional enableCheck               plugins.check
    ++ optional enableConvert             ffmpeg_3
    ++ optional enableDiscogs             pythonPackages.discogs_client
    ++ optional enableGmusic              pythonPackages.gmusicapi
    ++ optional enableKeyfinder           keyfinder-cli
    ++ optional enableLastfm              pythonPackages.pylast
    ++ optional enableMpd                 pythonPackages.mpd2
    ++ optional enableSonosUpdate         pythonPackages.soco
    ++ optional enableThumbnails          pythonPackages.pyxdg
    ++ optional enableWeb                 pythonPackages.flask
    ++ optional (enableWeb || enableZero) pythonPackages.flask
    ++ optional enableAlternatives        plugins.alternatives
    ++ optional enableCopyArtifacts       plugins.copyartifacts;
```